code/style.css

```diff
@@ -35,24 +94,94 @@ p {
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  position: relative;
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
   background: midnightblue;
-
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  position: relative;
 }
 
 button {
-  padding: 8px 16px;
-  background: crimson;
+  padding: 10px 18px;
+  background: black;
+  color: #f5d900;
   border: none;
-  font-size: 20px;
+  font-size: 25px;
   border-radius: 8px;
   display: block;
-  margin: 0 auto;
+  margin: 0 auto 30px auto;
+  font-family: 'Carter One', cursive;
+}
+
+#restartButton {
+  display: none;
+}
+
+article, #revealButton, #restartButton {
+  cursor: pointer;
 }
+
+#mystery {
+  width: 800px;
+  text-align: center;
+  margin: 10px auto;
+  font-size: 30px;
+}
```

Comment:
Remember, it's easy to use the `id`s for styling, but we use `id`s for javascript and classes for styling